.ci/Jenkinsfile

```diff
@@ -295,7 +295,14 @@ def withCloudTestEnv(Closure body) {
   }
   // Masking
   withEnvMask(vars: maskedVars) {
-    body()
+    // Env variables to be populated as tag/labels for the cloud resources
+    // NOTE: values can contain only lowercase letters, numeric characters, underscores, and dashes
+    withEnv(["TF_VAR_CREATED_DATE_TIME=${getCurrentBuildTime().toLowerCase().replaceAll(':', '_')}",
```

Comment:
The idea behind TF_VAR_ is to not leak it outside to Jenkinsfile and keep it inside the elastic-package logic. In this case, they would be `CREATED_DATE_TIME`, `BRANCH_NAME`, `ENVIRONMENT`, `REPO`.

Reply:
I used the `TF_VAR` prefix to help to identify whats' the purpose for those env variables, I could get rid of it, but I think it provides the context about who will consume it easily and it's harmless, what do you think?

Reply:
Actually, I would go with the other way round, Terraform framework is the implementation detail, and the framework should use available vars.

Reply:
I'm not 100% convinced, thought I'll change it